### PR TITLE
[ws-man-bridge] Refactor constructors and cleanup deps

### PR DESCRIPTION
**Pull request by easyCZ**
## Related Issue(s)

Fixes #

#### Preview status

gitpod:summary

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
</details>

/hold

### CODE REVIEW

File: components/ws-manager-bridge/src/bridge-controller.ts [hunk 1]
```diff
@@ -19,20 +19,13 @@ import { Metrics } from "./metrics";
...
-    @inject(Configuration)
-    protected readonly config: Configuration;
-
-    @inject(WorkspaceManagerBridgeFactory)
-    protected readonly bridgeFactory: interfaces.Factory<WorkspaceManagerBridge>;
-
-    @inject(WorkspaceManagerClientProvider)
-    protected readonly clientProvider: WorkspaceManagerClientProvider;
-
-    @inject(WorkspaceClusterDB)
```
Comment (easyCZ) [inline components/ws-manager-bridge/src/bridge-controller.ts:31]: removed

File: components/ws-manager-bridge/src/bridge-controller.ts [hunk 2]
```diff
@@ -19,20 +19,13 @@ import { Metrics } from "./metrics";
...
-    protected readonly metrics: Metrics;
+    constructor(
+        @inject(Configuration) private readonly config: Configuration,
+        @inject(WorkspaceManagerBridgeFactory)
+        private readonly bridgeFactory: interfaces.Factory<WorkspaceManagerBridge>,
+        @inject(WorkspaceManagerClientProvider) private readonly clientProvider: WorkspaceManagerClientProvider,
+        @inject(Metrics) private readonly metrics: Metrics,
+    ) {}
 
     protected readonly bridges: Map<string, WorkspaceManagerBridge> = new Map();
```
Comment (svenefftinge) [inline components/ws-manager-bridge/src/bridge-controller.ts:30]: We can also generally replace all usages of `protected` with `private` as we are not using inheritance usually. This often also reveals that methods or fields are actually unused.

**Review by svenefftinge — APPROVED** [review]
would be nice if you could also do the protected->private change with such cleanups

**Pull request merged**